src/base/misc/fatfs_boot.S

```diff
@@ -87,9 +87,27 @@ write_msg_20:
 	int	$0x10
 	jmp	write_msg
 
-/*# 5 magic bytes (for IO.SYS compatibility) :-))*/
-	.byte	4, 3, 2, 1, 13
-text:
+	.align 2
+	.globl	boot_txfr
+boot_txfr:
+	.word	0x9090 /* start ofs */
```

Comment:
Well, if you want a filler byte, use a define.
But don't use 0x90 as it falsely suggests NOPs.
Common filler patterns are: 00, ff, a5, bb etc.
You can also leave them uninitialized by using '?' IIRC.
Then with valgrind you may be able to detect
uninitialized reads.